### PR TITLE
fix(tether): print out a warning message if an anchor ref is not set

### PR DESCRIPTION
**Pull request by nadiia**

### CODE REVIEW

File: src/layer/tether.js [hunk 1]
```diff
@@ -43,7 +43,14 @@ class Tether extends React.Component<TetherPropsT, TetherStateT> {
         this.popper && this.popper.scheduleUpdate();
       }
       if (this.state.isMounted !== prevState.isMounted) {
-        this.initializePopper();
+        if (!this.props.anchorRef) {
+          // eslint-disable-next-line no-console
+          console.warn(
+            `[baseui][TetherBehavior] ref has not been passed to the Popper's anchor element`,
```
Comment (chasestarr) [inline src/layer/tether.js]: this lgtm. would it be helpful to provide a link to docs + example describing the problem?

Comment (gergelyke) [inline src/layer/tether.js]: should change the wording a bit, so it offers some advice on how to fix it?

Comment (chasestarr) [reply src/layer/tether.js]: i also wonder if this should be limited to `__DEV__`

Comment (nadiia) [reply src/layer/tether.js]: Good point! Yes, just `__DEV__`.

**Pull request merged**